infrastructure/tests/test_data_import.py

```diff
@@ -124,3 +187,111 @@ def test_year_error(self):
         self.assertEquals(AnnualSpendFile.objects.all().count(), 1)
         spend_file = AnnualSpendFile.objects.first()
         self.assertEquals(spend_file.status, AnnualSpendFile.ERROR)
+
+
+    def test_upload_project(self):
+        """Scope of Test: With no existing projects run an upload and check that the correct fields are populated"""
+        self.assertEquals(BudgetPhase.objects.all().count(), 5)
+
+        geography = Geography.objects.get(geo_code="BUF")
+        utils.load_file(geography, generate_mock_data(), "2019/2020")
+
+        project = Project.objects.get(function="Administrative and Corporate Support")
+        self.assertEquals(project.project_description, "P-CNIN FURN & OFF EQUIP")
+        self.assertEquals(project.project_number, "PC002003005_00002")
+        self.assertEquals(project.project_type, "New")
+        self.assertEquals(project.mtsf_service_outcome, "An efficient, effective and development-oriented public service")
+        self.assertEquals(project.iudf, "Growth")
+        self.assertEquals(project.own_strategic_objectives, "OWN MUNICIPAL STRATEGIC OBJECTIVE")
+        self.assertEquals(project.asset_class, "Furniture and Office Equipment")
+        self.assertEquals(project.asset_subclass, "")
+        self.assertEquals(project.ward_location, "Administrative or Head Office")
+        self.assertEquals(project.longitude, 0.0)
+        self.assertEquals(project.latitude, 0.0)
+
+        self.assertEquals(Expenditure.objects.all().count(), 5)
+        expenditure = Expenditure.objects.get(amount=340609.00)
+        self.assertEquals(expenditure.project.project_description, "P-CNIN FURN & OFF EQUIP")
+        self.assertEquals(expenditure.budget_phase.name, "Audited Outcome")
+        self.assertEquals(expenditure.financial_year.budget_year, "2017/2018")
```

Comment:
I think we need to be checking each year and phase here to assert that all of them are created

Reply:
when we do quarterly, we'll assert exactly which ones are created, and assert that no more than the ones checked are created, thus demonstrating that only a select set are created and specific ones are ignored without listing them explicitly.

Reply:
I made some changes to increase coverage of relevant expenditures, any expenditure that should update is checked